### PR TITLE
Create UI Structure

### PR DESCRIPTION
**Pull request by hannessolo**

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+import type {AppProps} from 'next/app'
+import {defaultTheme, Provider, SSRProvider} from "@adobe/react-spectrum";
 
-function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+function MyApp({Component, pageProps}: AppProps) {
+    return (
+        // SSR Provider is required for SSR/Static rendering
+        <SSRProvider>
+            <Provider theme={defaultTheme}>
```
Comment (icaraps) [inline pages/_app.tsx:9]: Can you pls use the light theme ? As the parent shell is also using the light theme by default

File: .gitignore [hunk 2]
```diff
@@ -33,3 +33,5 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+.idea
```
Comment (icaraps) [inline .gitignore:37]: This can be added in your global gitignore so it's not required here

File: Slides/SummarySlide.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import SlideLayout from "../components/SlideLayout";
+import {useContext} from "react";
+import FormContext from "../logic/FormContext";
+import Nav from "../components/Nav";
+
+export default function SummarySlide() {
+  const {next, prev, formData} = useContext(FormContext)
+
+  return <SlideLayout>
+    <h1>Ok, thanks User</h1>
```
Comment (icaraps) [inline Slides/SummarySlide.tsx:10]: Can you use spectrum's typography instead here ? You'll have to import the css package https://opensource.adobe.com/spectrum-css/typography.html 
We'll also add the adobe font later.

File: Slides/SummarySlide.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <span>Here’s what we will record for your trial:</span>
```
Comment (icaraps) [inline Slides/SummarySlide.tsx:11]: Can you use a layout component here to display the data ?

File: components/Slides.module.css [hunk 5]
```diff
@@ -0,0 +1,31 @@
+.actionButtonContainer {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+}
+
+.actionButtonContainer > * {
```
Comment (icaraps) [inline components/Slides.module.css:7]: Maybe you can be more specific in your selector here

**Review by icaraps — CHANGES_REQUESTED** [review]
LGTM ! There's one thing major I'd like to change if possible. Could you think of a way to extract the raw data (questions, responses, text) into another file (eg json) to separate the layout from the actual data ? 
I also noticed a few spacing issues eg on slide 2 between the select and the buttons. Or alignment issues, eg radio boxes are not vertically aligned. 
In general, I think it would make sense to increase the container width and add more spacing while making sure it's responsive.

File: Slides/SummarySlide.tsx (hunk 3 above)
Comment (hannessolo) [reply Slides/SummarySlide.tsx:11]: I'm not sure how to do this, I don't think I can import that library into nextjs?

**Comment by hannessolo** [comment]
- I improved the layout somewhat, but will wait before doing more in case Ron decides to change it, and focus on functionality first
- I'm not sure how to import the css typography library you commented
- I extracted the long strings, but won't do smaller ones yet, because I have a branch where I changed some of these - so I don't have to do it all twice
- I will move the repo to git corp once we merge this

**Review by icaraps — APPROVED** [review]

**Pull request merged**